jetty-ee10/jetty-ee10-osgi/jetty-ee10-osgi-boot-jsp/src/main/java/org/eclipse/jetty/ee10/osgi/boot/jasper/ContainerTldBundleDiscoverer.java

```diff
@@ -224,15 +224,15 @@ private void convertBundleLocationToURL(BundleFileLocatorHelper locatorHelper, B
         {
             for (File f : jasperLocation.listFiles())
             {
-                if (f.getName().endsWith(".jar") && f.isFile())
+                if (FileID.isJavaArchvie(f.getName()) && f.isFile())
```

Comment:
Typo: `isJavaArchvie`.

Reply:
Done (FYI, this was on code that is in a project that is not being compiled)